testing/cucumber/features/linters/unique_scenario_names.feature

```diff
@@ -0,0 +1,71 @@
+Feature: Unique scenario names linter
+
+  As a tester
+  I want each scenario to have a unique name even though they were created using a template.
+  So that I can easily identify and refer to them without confusion
+
+  Scenario: Detect scenarios with duplicate names within the same feature file
```

Comment:
I suggest to use more meaningful names of the scenarios.

First, look at the neighboring linters. Secondly, the idea is to show in one sentence (title of scenario) what's inside. Inside: if two names are the same, the linter gives an error.

Should be fixed here and everywhere

Reply:
The neighboring tests have very short scenario names, like:

```
Scenario: Linting (Good)
Scenario: Linting (Bad)
```

I added a little to those descriptions:

```
Scenario: Linting (Duplicates within Feature)
Scenario: Linting (Duplicates from Scenario Outline)
```

Reply:
I suggest a mix of it:
```
Scenario: Linting (Good: No Duplicates within the Feature)
Scenario: Linting (Bad: Duplicates from Scenario Outline)
```

Reply:
Done